chasm/test_component_test.go

```diff
@@ -95,9 +102,9 @@ func (tc *TestComponent) Fail(_ MutableContext) {
 }
 
 // SearchAttributes implements VisibilitySearchAttributesProvider interface.
-func (tc *TestComponent) SearchAttributes(_ Context) map[string]VisibilityValue {
-	return map[string]VisibilityValue{
-		testComponentStartTimeSAKey: VisibilityValueTime(tc.ComponentData.GetStartTime().AsTime()),
+func (tc *TestComponent) SearchAttributes(_ Context) []SearchAttributeKeyValue {
+	return []SearchAttributeKeyValue{
+		testComponentStartTimeSearchAttribute.Value(tc.ComponentData.GetStartTime().AsTime()),
```

Comment:
Can you add a search attribute that is mapped to an existing non-chasm pre-defined field for completeness of this test?